python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -449,18 +425,88 @@ def measureMeanVarCov(self, exposure1, exposure2, region=None, covAstierRealSpac
         wDiff = np.where(diffIm.getMask().getArray() == 0, 1, 0)
         w = w12*wDiff
 
-        if np.sum(w) < self.config.minNumberGoodPixelsForFft:
+        if np.sum(w) < self.config.minNumberGoodPixelsForFastFourierTransform:
```

Comment:
Does this mean the real space/direct covariance calculation also can't be constructed?  If so, then could this be `minNumberPixelsForCovariance`?  Otherwise, it seems like this should warn, and then attempt to use the direct calculation.

Reply:
Right (to the first option). This `w` weight matrix is also an input of the function that calculates the covariance in real space, so I'll change the name to your suggestion.